packages/flutter/lib/src/widgets/text_editing_action.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart' show RenderEditable;
+
+import 'actions.dart';
+import 'editable_text.dart';
+import 'focus_manager.dart';
+import 'framework.dart';
+
+/// An implementor of this must be focused for a [TextEditingAction] to be
+/// enabled.
+///
+/// See also:
+///
+///   * [EditableText], which implements this and is the most typical target of
+///     a TextEditingAction.
+abstract class TextEditingActionTarget {
```

Comment:
Ah I see what you mean.  I like this idea, but do you mind if I punt on it until I've converted more stuff to actions?  I'm still not fully certain on the long-term role of EditableText and RenderEditable.  I might even want to create a separate class from both of these that would implement all of this text editing API instead of RenderEditable.

That would be a breaking change, but it's probably unavoidable.